pyfakefs/fake_filesystem_unittest.py

```diff
@@ -88,6 +90,9 @@ def __init__(self, methodName='runTest'):
         super(TestCase, self).__init__(methodName)
         if additional_skip_names is not None:
             Patcher.SKIPNAMES.update(additional_skip_names)
+        if not patch_path:
+            Patcher.patch_path = True
+            Patcher.SKIPNAMES.remove('path')
```

Comment:
Yes, I didn't like this either. I will have another go at this tomorrow, if I find the time,  or next week.